data/translation/Zed/de_DE.csv

```diff
@@ -55,7 +55,7 @@ Default Business Unit,Standardgeschäftsbereich
 base64,base64
 url-encoded,url-encoded
 No Encoding,Keine Kodierung
-Request Mapping,Mapping anfragen
+Request Mapping,Mapping der Anfrage
```

Comment:
Please check this translation.
```suggestion
Request Mapping,Mapping des Anfrage
```

Reply:
Done according to https://spryker.atlassian.net/browse/PS-9734?focusedCommentId=76638&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-76638